config/web/settings.ini

```diff
@@ -72,6 +72,10 @@ web.https.cyphers.excluded.add=TLS_DHE.*,TLS_EDH.*
 # web.expect_ct=
 # web.referrer_policy=strict-origin-when-cross-origin
 
+# Optional configuration that will enable a Content-Security-Policy or Content-Security-Policy-Report-Only response headers
+# web.content.security.policy=
+# web.content.security.policy.report.only=
```

Comment:
Is it possible to have sample values (or the enumeration of possible values) for both properties ?

Reply:
it is not a boolean, but I can add basic configuration examples